velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -90,12 +90,16 @@ int64_t rand(FuzzerGenerator& rng) {
 
 template <>
 double rand(FuzzerGenerator& rng) {
-  return boost::random::uniform_01<double>()(rng);
+  auto d = boost::random::uniform_01<double>()(rng);
```

Comment:
I made this change to ensure I got good coverage on type of doubles generated. I am not sure if other fuzzers break due to this change. I will remove it if other fuzzers detect issues due to this.